openid-4-verifiable-presentations-1_0.md

```diff
@@ -706,6 +724,20 @@ The Wallet then validates the request as specified in OAuth 2.0 [@RFC6749].
 
 If the Verifier responds with any HTTP error response, the Wallet MUST terminate the process.
 
+## Attachments in Authorization Requests {#attachments-in-authorization-requests}
+
+This section provides guidance on processing and security considerations for the `attachments` parameter defined in (#new_parameters).
+
+### Proof of Possession
+
+In some cases, the Verifier may want to include attachments that are bound to a key or require verification of their origin.
+
+It is up to the ecosystem profile to define how proof of possession is conveyed. Verifiers SHOULD use the `nonce` parameter from the Authorization Request in any proof-of-possession mechanism to bind to this transaction. The Wallet MUST validate such proofs if defined by the profile and ignore or reject attachments that fail validation.
```

Comment:
Having thought about this more I'm not convinced nonce is sufficient to bind to the transaction. I think you need at least client id embedded in to the proof of possessions too.

Reply:
including the client id reduces the risk of the replay attack. I would prefer to add some suggestions to build the proof than leaving it open to the implementations, risking vulnerabilities.

Reply:
Added the `client_id` as a parameter.